pkgs/development/python-modules/python-hl7/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "python-hl7";
+  version = "0.4.5";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Please use this new attribute